draft-ietf-ohai-ohttp.md

```diff
@@ -726,6 +726,12 @@ In this section, a deployment where there are three entities is considered:
 * A relay operates the Oblivious Relay Resource
 * A server operates both the Oblivious Gateway Resource and the Target Resource
 
+Connections between the client, Oblvious Relay Resource, and Oblivious Gateway
+Resource MUST use HTTPS in order to provide unlinkability in the presence of a
+network observer.  The scheme of the encapsulated request detemrines what is
```

Comment:
In case it's helpful, the analysis showed that HTTPS is only needed between client and relay to achieve this property. But we're mandating it on both sides because it's just good practice.